### DISCUSSION TITLE
tiledlayout(3, 1) dimensions of layout of the window

### DISCUSSION THREAD

**Discussion by vlsireddy** [alandefreitas/matplotplusplus#249] [Q&A] [answered]
Hi,

How to set the dimensions of the entire tiledlayout winow. 

what is the function.

Can you plz point to an example.

Thanks

**Comment by alandefreitas** [comment, marked as answer]
You probably want subplots instead:

https://alandefreitas.github.io/matplotplusplus/appearance/multiplot/subplots/

**Comment by vlsireddy** [comment]
ok, i got this working. Thanks for reply.
